pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, stdenv, fetchurl, writeScript, runtimeShell, makeFontsConf,
+  p7zip, autoPatchelfHook, desktop-file-utils, imagemagick, makeWrapper,
+  twemoji-color-font, xorg, libsodium, libopus, libGL, zlib, qt5, alsaLib }:
+
+stdenv.mkDerivation rec {
+  pname = "ripcord";
+  version = "0.4.23";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/Ripcord-${version}-x86_64.AppImage";
+    sha256 = "0395w0pwr1cz8ichcbyrsscmm2p7srgjk4vkqvqgwyx41prm0x2h";
+    name = "${pname}-${version}.AppImage";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook p7zip desktop-file-utils imagemagick makeWrapper ];
+  buildInputs = [ libsodium libopus libGL alsaLib ] ++
+                (with xorg; [ libX11 libXScrnSaver libXcursor xkeyboardconfig ]) ++
+                (with qt5; [ qtbase qtsvg qtmultimedia qtwebsockets ]);
```

Comment:
don't do `with qt5`. With the `qt5.callPackage` these will all be in scope at the proper version..